### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by ericcurtin**
Fix GitHub yaml

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,8 +20,9 @@ jobs:
       - uses: actions/checkout@v4
       - name: install golang
         shell: bash
-        run: brew install go
-	run: make install-requirements
+        run: |
+           brew install go
+           make install-requirements
```
Comment (rhatdan) [inline .github/workflows/ci.yml:25]: This can only be done on Linux right now?

**Comment by ericcurtin** [comment]
@rhatdan when github yaml is wrong, it doesn't trigger CI at all, it's happened a few times, so all builds will be green as CI hasn't been run

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ericcurtin) [reply .github/workflows/ci.yml:25]: Not if we use pipx...

Ubuntu 24.04 and macOS bail when using pip and recommend pipx

**Comment by ericcurtin** [comment]
@rhatdan you might be better off taking up this PR, you are most familiar with the recent changes

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ericcurtin) [reply .github/workflows/ci.yml:25]: This is why CI stopped running FWIW:

run:

followed by

run:

**Pull request closed without merging**